plugins/template.md

```diff
@@ -88,16 +102,26 @@ tinymce.init({
   plugins: "template",
   menubar: "insert",
   toolbar: "template",
-  template_cdate_classes: "cdate creationdate"
+  template_cdate_format: "%m/%d/%Y : %H:%M",
```

Comment:
I think this should go in the section below? This setting is for classes, not format

Reply:
fixed